Packages.props

```diff
@@ -2,32 +2,32 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <!-- Solution dependencies -->
-    <PackageReference Update="AdysTech.CredentialManager" Version="2.1.1" />
-    <PackageReference Update="Atlassian.SDK" Version="12.1.1" />
-    <PackageReference Update="AppInsights.WindowsDesktop" Version="2.13.1" />
-    <PackageReference Update="Ben.Demystifier" Version="0.1.6" />
-    <PackageReference Update="ConEmu.Core" Version="21.03.14" />
-    <PackageReference Update="EnvDTE" Version="16.9.31023.347" />
+    <PackageReference Update="AdysTech.CredentialManager" Version="2.3.0" />
+    <PackageReference Update="Atlassian.SDK" Version="12.4.0" />
+    <PackageReference Update="AppInsights.WindowsDesktop" Version="2.17.10" />
+    <PackageReference Update="Ben.Demystifier" Version="0.4.1" />
+    <PackageReference Update="ConEmu.Core" Version="21.7.18" />
+    <PackageReference Update="EnvDTE" Version="16.10.31320.204" />
     <PackageReference Update="ExCSS" Version="4.1.0" />
-    <PackageReference Update="GitInfo" Version="2.0.21" />
-    <PackageReference Update="JetBrains.Annotations" Version="2020.1.0" />
+    <PackageReference Update="GitInfo" Version="2.1.2" />
+    <PackageReference Update="JetBrains.Annotations" Version="2021.2.0" />
     <PackageReference Update="LibGit2Sharp" Version="0.26.2" />
     <PackageReference Update="Microsoft.Toolkit.HighPerformance" Version="7.0.2" />
-    <PackageReference Update="Microsoft.VisualStudio.Composition" Version="16.4.11" />
-    <PackageReference Update="Microsoft.VisualStudio.Threading" Version="16.7.56" />
+    <PackageReference Update="Microsoft.VisualStudio.Composition" Version="16.9.20" />
+    <PackageReference Update="Microsoft.VisualStudio.Threading" Version="16.10.56" />
     <PackageReference Update="Microsoft-WindowsAPICodePack-Core" Version="1.1.4" />
     <PackageReference Update="Microsoft-WindowsAPICodePack-Shell" Version="1.1.4" />
-    <PackageReference Update="Newtonsoft.Json" Version="12.0.3" />
-    <PackageReference Update="NString" Version="1.2.0" />
-    <PackageReference Update="RestSharp" Version="106.11.4" />
-    <PackageReference Update="SmartFormat.NET" Version="2.5.0" />
+    <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Update="NString" Version="2.1.1" />
+    <PackageReference Update="RestSharp" Version="106.12.0" />
+    <PackageReference Update="SmartFormat.NET" Version="2.7.0" />
     <PackageReference Update="StyleCop.Analyzers" Version="1.2.0-beta.321" />
```

Comment:
Are the `StyleCop.Analyzers` really obsolete? Do we need to migrate the regarding settings?